.github/workflows/ci-app.yml

```diff
@@ -1,4 +1,4 @@
-name: Template API Checks
+name: CI - App
 
 on:
   push:
```

Comment:
Are the `paths` below correct? I'm wondering if they didn't get updated if/when this filename was changed

Reply:
good catch will fix